rust/noosphere-ns/src/dht/processor.rs

```diff
@@ -350,7 +348,13 @@ impl DHTProcessor {
                 } => {}
                 kad::InboundRequest::PutRecord { source, record, .. } => match record {
                     Some(rec) => {
-                        if self.swarm.behaviour_mut().kad.store_mut().put(rec.clone()).is_err()
+                        if self
+                            .swarm
+                            .behaviour_mut()
+                            .kad
+                            .store_mut()
+                            .put(rec.clone())
+                            .is_err()
```

Comment:
I'll add a log, these are errors failing to save some data from another node which can only occur for invalid data, a full store, or a data that's too that generally feels more like pruning rather than an error-error

Reply:
Oh i see, the error isn't in the log, nice catch!